2017-01-05 16:04:39.149 ERROR [ServerInfoTimer] com.haulmont.cuba.core.app.ServerInfo - Unable to update SYS_SERVER: {}
java.sql.SQLException: Network error IOException: Network is unreachable (connect failed)
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:395) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.ConnectionJDBC3.<init>(ConnectionJDBC3.java:50) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.Driver.connect(Driver.java:184) ~[jtds-1.2.4.jar:1.2.4]
	at org.apache.tomcat.dbcp.dbcp2.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:255) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.create(GenericObjectPool.java:867) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:432) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:360) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.PoolingDataSource.getConnection(PoolingDataSource.java:133) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSource.getConnection(BasicDataSource.java:1532) ~[tomcat-dbcp.jar:8.0.35]
	at com.haulmont.bali.db.QueryRunner.prepareConnection(QueryRunner.java:258) ~[cuba-global-6.3.5.jar:6.3.5]
	at com.haulmont.bali.db.QueryRunner.update(QueryRunner.java:627) ~[cuba-global-6.3.5.jar:6.3.5]
	at com.haulmont.cuba.core.app.ServerInfo.updateCurrentServer(ServerInfo.java:177) ~[cuba-core-6.3.5.jar:6.3.5]
	at com.haulmont.cuba.core.app.ServerInfo$1.run(ServerInfo.java:129) [cuba-core-6.3.5.jar:6.3.5]
	at java.util.TimerThread.mainLoop(Timer.java:555) [na:1.8.0_112]
	at java.util.TimerThread.run(Timer.java:505) [na:1.8.0_112]
Caused by: java.net.SocketException: Network is unreachable (connect failed)
	at java.net.PlainSocketImpl.socketConnect(Native Method) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_112]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_112]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_112]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_112]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_112]
	at net.sourceforge.jtds.jdbc.SharedSocket.createSocketForJDBC3(SharedSocket.java:305) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.SharedSocket.<init>(SharedSocket.java:255) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:323) ~[jtds-1.2.4.jar:1.2.4]
	... 15 common frames omitted
2017-01-05 16:04:39.179 ERROR [ServerInfoTimer] com.haulmont.cuba.core.app.ServerInfo - Unable to update SYS_SERVER: {}
java.sql.SQLException: Network error IOException: Network is unreachable (connect failed)
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:395) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.ConnectionJDBC3.<init>(ConnectionJDBC3.java:50) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.Driver.connect(Driver.java:184) ~[jtds-1.2.4.jar:1.2.4]
	at org.apache.tomcat.dbcp.dbcp2.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:255) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.create(GenericObjectPool.java:867) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:432) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:360) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.PoolingDataSource.getConnection(PoolingDataSource.java:133) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSource.getConnection(BasicDataSource.java:1532) ~[tomcat-dbcp.jar:8.0.35]
	at com.haulmont.bali.db.QueryRunner.prepareConnection(QueryRunner.java:258) ~[cuba-global-6.3.5.jar:6.3.5]
	at com.haulmont.bali.db.QueryRunner.update(QueryRunner.java:627) ~[cuba-global-6.3.5.jar:6.3.5]
	at com.haulmont.cuba.core.app.ServerInfo.updateCurrentServer(ServerInfo.java:177) ~[cuba-core-6.3.5.jar:6.3.5]
	at com.haulmont.cuba.core.app.ServerInfo$1.run(ServerInfo.java:129) [cuba-core-6.3.5.jar:6.3.5]
	at java.util.TimerThread.mainLoop(Timer.java:555) [na:1.8.0_112]
	at java.util.TimerThread.run(Timer.java:505) [na:1.8.0_112]
Caused by: java.net.SocketException: Network is unreachable (connect failed)
	at java.net.PlainSocketImpl.socketConnect(Native Method) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_112]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_112]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_112]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_112]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_112]
	at net.sourceforge.jtds.jdbc.SharedSocket.createSocketForJDBC3(SharedSocket.java:305) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.SharedSocket.<init>(SharedSocket.java:255) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:323) ~[jtds-1.2.4.jar:1.2.4]
	... 15 common frames omitted
2017-01-05 17:16:14.108 ERROR [ServerInfoTimer] com.haulmont.cuba.core.app.ServerInfo - Unable to update SYS_SERVER: {}
java.sql.SQLException: Network error IOException: Operation timed out (Connection timed out)
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:395) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.ConnectionJDBC3.<init>(ConnectionJDBC3.java:50) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.Driver.connect(Driver.java:184) ~[jtds-1.2.4.jar:1.2.4]
	at org.apache.tomcat.dbcp.dbcp2.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:255) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.create(GenericObjectPool.java:867) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:432) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:360) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.PoolingDataSource.getConnection(PoolingDataSource.java:133) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSource.getConnection(BasicDataSource.java:1532) ~[tomcat-dbcp.jar:8.0.35]
	at com.haulmont.bali.db.QueryRunner.prepareConnection(QueryRunner.java:258) ~[cuba-global-6.3.5.jar:6.3.5]
	at com.haulmont.bali.db.QueryRunner.update(QueryRunner.java:627) ~[cuba-global-6.3.5.jar:6.3.5]
	at com.haulmont.cuba.core.app.ServerInfo.updateCurrentServer(ServerInfo.java:177) ~[cuba-core-6.3.5.jar:6.3.5]
	at com.haulmont.cuba.core.app.ServerInfo$1.run(ServerInfo.java:129) [cuba-core-6.3.5.jar:6.3.5]
	at java.util.TimerThread.mainLoop(Timer.java:555) [na:1.8.0_112]
	at java.util.TimerThread.run(Timer.java:505) [na:1.8.0_112]
Caused by: java.net.ConnectException: Operation timed out (Connection timed out)
	at java.net.PlainSocketImpl.socketConnect(Native Method) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_112]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_112]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_112]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_112]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_112]
	at net.sourceforge.jtds.jdbc.SharedSocket.createSocketForJDBC3(SharedSocket.java:305) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.SharedSocket.<init>(SharedSocket.java:255) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:323) ~[jtds-1.2.4.jar:1.2.4]
	... 15 common frames omitted
2017-01-05 17:16:14.113 ERROR [ServerInfoTimer] com.haulmont.cuba.core.app.ServerInfo - Unable to update SYS_SERVER: {}
java.sql.SQLException: Network error IOException: Network is unreachable (connect failed)
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:395) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.ConnectionJDBC3.<init>(ConnectionJDBC3.java:50) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.Driver.connect(Driver.java:184) ~[jtds-1.2.4.jar:1.2.4]
	at org.apache.tomcat.dbcp.dbcp2.DriverConnectionFactory.createConnection(DriverConnectionFactory.java:38) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:255) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.create(GenericObjectPool.java:867) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:432) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:360) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.PoolingDataSource.getConnection(PoolingDataSource.java:133) ~[tomcat-dbcp.jar:8.0.35]
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSource.getConnection(BasicDataSource.java:1532) ~[tomcat-dbcp.jar:8.0.35]
	at com.haulmont.bali.db.QueryRunner.prepareConnection(QueryRunner.java:258) ~[cuba-global-6.3.5.jar:6.3.5]
	at com.haulmont.bali.db.QueryRunner.update(QueryRunner.java:627) ~[cuba-global-6.3.5.jar:6.3.5]
	at com.haulmont.cuba.core.app.ServerInfo.updateCurrentServer(ServerInfo.java:177) ~[cuba-core-6.3.5.jar:6.3.5]
	at com.haulmont.cuba.core.app.ServerInfo$1.run(ServerInfo.java:129) [cuba-core-6.3.5.jar:6.3.5]
	at java.util.TimerThread.mainLoop(Timer.java:555) [na:1.8.0_112]
	at java.util.TimerThread.run(Timer.java:505) [na:1.8.0_112]
Caused by: java.net.SocketException: Network is unreachable (connect failed)
	at java.net.PlainSocketImpl.socketConnect(Native Method) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) ~[na:1.8.0_112]
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) ~[na:1.8.0_112]
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) ~[na:1.8.0_112]
	at java.net.Socket.connect(Socket.java:589) ~[na:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_112]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_112]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_112]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_112]
	at net.sourceforge.jtds.jdbc.SharedSocket.createSocketForJDBC3(SharedSocket.java:305) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.SharedSocket.<init>(SharedSocket.java:255) ~[jtds-1.2.4.jar:1.2.4]
	at net.sourceforge.jtds.jdbc.ConnectionJDBC2.<init>(ConnectionJDBC2.java:323) ~[jtds-1.2.4.jar:1.2.4]
	... 15 common frames omitted
